lib/ansible/modules/packaging/language/pip.py

```diff
@@ -509,6 +529,15 @@ def main():
                     has_vcs = True
                     break
 
+        # check invalid combination of arguments
+        if version is not None:
+            if len(name) > 1:
```

Comment:
This check is invalid because strings are iterable sequences.
`len('CherryPy') == 8`, `len(['aiohttp', 'aiohttp_jinja2']) == 2`.

Please use some of:
```python
from ansible.module_utils.common._collections_compat import is_sequence
...
is_sequence(..., include_strings=False)
```

or

```python
from ansible.module_utils.common._collections_compat import is_string
...
not is_string(...)
```

Reply:
I think the module guarantee that name must be a list. But yes just `len(name)` is ambiguous and it's better to use `is_sequence`.